src/routes/blog/post/what-is-mcp/+page.markdoc

```diff
@@ -0,0 +1,210 @@
+
+An employee could just ask:
+
+- *“What were last month's top-selling products?”*
+- *“Are we seeing any patterns in customer complaints?”*
+
+And instead of getting **generic answers**, the AI pulls **real company data** and responds with **accurate insights**.
+
+## 3. A real AI personal assistant
+
+Imagine having an AI assistant that actually understands your files, emails, and calendar.
+
+Instead of you manually searching for an old PDF or checking your schedule, you just ask:
+
+- *“Summarize my meeting notes from last week.”*
+- *“Find the invoice I received in February.”*
+
+MCP lets AI assistants **fetch real information** without you needing to switch apps or dig through folders.
+
+## 4. Cloud platforms like Appwrite
+
+Appwrite has introduced the **Appwrite MCP server** to allow AI assistants to interact with Appwrite projects directly. With this, you can now **query databases, fetch user lists, and trigger actions** on your Appwrite project using AI assistants.
+
+# More about Appwrite MCP server
+
+Appwrite MCP is an **MCP server** that connects AI assistants to your **Appwrite project**.
+
+It currently supports:
+
+- **Databases**: Query and retrieve database records.
+- **Users**: Fetch user lists and details.
+
+## How it works
+
+Here's a simple example of how Appwrite MCP works:
+
+1. **The AI assistant sends a request** via MCP (e.g., *“Get all users from my Appwrite project”*).
+2. **The Appwrite MCP server receives the request** and translates it into an Appwrite API call.
+3. **The Appwrite API returns the data** to the MCP server.
+4. **The MCP server structures the response** in a format the AI can understand.
+5. **The AI assistant uses the data** to generate a response.
+
+## How to set it up
+
+An easy way to try out the Appwrite MCP is by using Claude Desktop. Install Claude on your computer if you don't have it already, then navigate to Settings → developer and click “Edit Config”. This should lead you to your `claude_desktop_config.json` file.
+
+Next, edit your config file and add this:
+
+```jsx
+"MCPServers": {
+  "appwrite": {
+    "command": "uvx",
+    "args": ["mcp-server-appwrite"],
+    "env": {
+      "APPWRITE_PROJECT_ID": "your-project-id",
+      "APPWRITE_API_KEY": "your-api-key"
+    }
+  }
+}
+```
+
+You'll need **uvx** on your computer for the command to work. Use this [link](https://docs.astral.sh/uv/getting-started/installation/) to install it.
+Next, opening Claude Desktop should show you a tool icon in the chat input field with a list of the available MCP tools.
+
+![Claude Desktop MCP tool icon](/images/blog/what-is-mcp/claude-mcp-tools.png)
+
+If you get an error that the Appwrite MCP server failed to start, it might be that your computer hasn't recognized the `uvx` command yet. In that case, you can replace the “uvx” string in your config object's `command` property with the direct path to your uv installation. Here's an example for mac:
```

Comment:
```suggestion
If you get an error that the Appwrite MCP server failed to start, it might be that your computer hasn't recognized the `uvx` command yet. In that case, you can replace the `uvx` string in your config object's `command` property with the direct path to your uv installation. Here's an example for MacOS:
```

Reply:
Also, it might make sense to add a PowerShell option too